Bounding_volumes/include/CGAL/min_quadrilateral_2.h

```diff
@@ -18,7 +18,7 @@
 
 
 #include <CGAL/basic.h>
-#include <CGAL/Optimisation/assertions.h>
```

Comment:
Here is the change that impacted the dependencies. That is why the CI said:
```
 Differences in Optimal_bounding_box: Optimisation_basic have disappeared.
 Differences in Matrix_search: Optimisation_basic have disappeared.
 Differences in Inscribed_areas: Optimisation_basic have disappeared.
```

That means that `Optimisation_basic` is no longer a dependency of those three packages before of the removals of `#include <CGAL/Optimisation/assertions.h>` in various files.

For the other message from the CI:
```
Differences in Optimisation_basic: Installation
Kernel_23
Profiling_tools
STL_Extension have disappeared.
```
I am still trying to figure out.